src/aca_main.cpp

```diff
@@ -83,13 +83,21 @@ static void aca_cleanup()
   }
 
   if (g_grpc_server != NULL) {
+    g_grpc_server->ShutDownServer();
     delete g_grpc_server;
     g_grpc_server = NULL;
     ACA_LOG_INFO("Cleaned up grpc server.\n");
   } else {
     ACA_LOG_ERROR("Unable to call delete, grpc server pointer is null.\n");
   }
 
+  if (g_grpc_server_thread != NULL){
+    delete g_grpc_server_thread;
+    g_grpc_server_thread = NULL;
+    ACA_LOG_INFO("Cleaned up grpc server thread.\n");
+  } else {
+    ACA_LOG_ERROR("Unable to call delete,grpc server thread pointer is null.\n");
```

Comment:
to be consistant with the rest of the comments, please change "delete,grpc" to "delete, grpc" (just add a space before the ,)